src/tools/debugger/entities/SteppingType.java

```diff
@@ -104,11 +110,37 @@ public void process(final Suspension susp) {
     }
   },
 
+  @SerializedName("stepToMessageRcvr")
+  STEP_TO_RECEIVER_MESSAGE("stepToMessageRcvr", "Step to Msg Receiver", Group.ACTOR_STEPPING, "msg-open", new Class[] {EventualMessageSend.class}, new ActivityType[] {ActivityType.ACTOR}) {
```

Comment:
I think we need to remove `new ActivityType[] {ActivityType.ACTOR}`, otherwise, you can't step from a `<-:` thats done by a process or thread to the receiver.